open-metadata-implementation/server-chassis/server-chassis-spring/pom.xml

```diff
@@ -66,6 +66,30 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-lineage-janus-connector</artifactId>
```

Comment:
Do we need this & the elasticsearch connectors as dependencies?

They tend to result in a significant chain of other dependencies which then everyone pulls in

(Note also - whatever we end up with should be mirrored in build.gradle)

Reply:
Indeed! I think they should be loaded with loader.path

Reply:
Removed them from the pom, my bad for including this module as dependecy